Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -854,15 +861,49 @@ void ExecuteCommand(ReplyType reply_type)
     case 0x00:  // Read Sector
     {
       const u64 dvd_offset = static_cast<u64>(s_DICMDBUF[1]) << 2;
-
       INFO_LOG_FMT(
           DVDINTERFACE,
           "Read: DVDOffset={:08x}, DMABuffer = {:08x}, SrcLength = {:08x}, DMALength = {:08x}",
           dvd_offset, s_DIMAR, s_DICMDBUF[2], s_DILENGTH);
 
       if (s_drive_state == DriveState::ReadyNoReadsMade)
         SetDriveState(DriveState::Ready);
-
+      if (DVDThread::GetDiscType() == DiscIO::Platform::Triforce)
+      {
+        if ((dvd_offset & 0x80000000) != 0)
+        {
+          switch (dvd_offset)
+          {
+          // Media board status (1)
+          case 0x80000000:
+            Memory::Memset(s_DIMAR, 0, s_DICMDBUF[2]);
+            break;
+          // Media board status (2)
+          case 0x80000020:
+            Memory::Memset(s_DIMAR, 0, s_DICMDBUF[2]);
+            break;
+          // Media board status (3)
+          case 0x80000040:
+            Memory::Memset(s_DIMAR, 0xFF, s_DICMDBUF[2]);
+            // DIMM size
+            Memory::Write_U32(0x20, s_DIMAR);
+            // GCAM signature
+            Memory::Write_U32(0x4743414D, s_DIMAR + 4);
+            break;
+          // Firmware status (1)
+          case 0x80000120:
+            Memory::Memset(s_DIMAR, 0x01, s_DICMDBUF[2]);
+            break;
+          // Firmware status (2)
+          case 0x80000140:
+            Memory::Memset(s_DIMAR, 0x01, s_DICMDBUF[2]);
+            break;
+          default:
+            PanicAlertFmtT("Unknown Media Board Read");
```

Comment:
```suggestion
            PanicAlertFmtT("Unknown Media Board Read");
```

Neither users nor translators will be able to make much sense of this string.

Reply:
Fair enough, although I don't think it should be a panic alert now that I think about it.